packages/agoric-cli/tools/getting-started.js

```diff
@@ -253,18 +160,9 @@ export const gettingStartedWorkflowTest = async (t, options = {}) => {
     t.is(
       await Promise.race([uiStartP, uiListening.promise]),
       'listening',
-      `cd ui && yarn start succeeded`,
+      `yarn start:ui succeeded`,
     );
     clearInterval(ival);
-
-    // Test that the Node.js `-r esm`-dependent plugin works.
```

Comment:
_remark_

I don't know what these ESM plugin tests are about so I removed them

Reply:
I think we can forget about resm-plugin